src/08_example/03_combinators.md

```diff
@@ -0,0 +1,27 @@
+# Multiple Concurrent Actions Per Request
```

Comment:
@betamos do you think it makes sense to get rid of this section? After addressing your comments I don't think it adds much.

Reply:
SGTM, given that you cover parallelism in the next section. Regarding the off-tasks of writing to databases etc, I'd like to discuss design patterns for that anyway. If you save the code we can take it separately, which would help this PR merge quicker.